src/Diagnostics.Logger/ETWProvider/DiagnosticsETWProvider.cs

```diff
@@ -140,6 +140,16 @@ public void LogRuntimeHostDetectorAscInsight(string RequestId, string Source, st
                 Details);
         }
 
+        [Event(2008, Level = EventLevel.Informational, Channel = EventChannel.Admin, Message = ETWMessageTemplates.LogRuntimeHostDetectorAscInsight)]
```

Comment:
Should this be LogRuntimeHostSupportTopicAscInsight ?

Reply:
yes good catch, though I am not even sure these are really being used.